autogluon/task/tabular_prediction/presets_configs.py

```diff
@@ -0,0 +1,68 @@
+preset_dict = dict(
+    # Best quality with little consideration to inference time or disk usage. Achieve even better results by specifying a large time_limits value.
+    # Recommended for applications that benefit from the best possible model quality.
+    best_quality={'auto_stack': True},
+
+    # Identical to best_quality but additionally trains refit_full models that have slightly lower quality but are over 10x faster during inference and require 10x less disk space.
+    best_quality_with_high_quality_refit={'auto_stack': True, 'refit_full': True},
+
+    # High quality with fast inference. ~10x-200x faster inference and ~10x-200x lower disk usage than `best_quality`.
+    # Recommended for applications that require reasonable inference speed and/or model size.
+    high_quality_fast_inference_only_refit={'auto_stack': True, 'refit_full': True, 'set_best_to_refit_full': True, 'save_bagged_folds': False},
+
+    # Good quality with very fast inference. ~4x faster inference and ~4x lower disk usage than `high_quality_fast_inference_only_refit`.
+    # Recommended for applications that require fast inference speed.
+    good_quality_faster_inference_only_refit={'auto_stack': True, 'refit_full': True, 'set_best_to_refit_full': True, 'save_bagged_folds': False, 'hyperparameters': 'light'},
+
+    # Medium quality with very fast inference and very fast training time. ~20x faster training than `good_quality_faster_inference_only_refit`.
+    # This is the default preset in AutoGluon, but should generally only be used for quick prototyping, as `good_quality_faster_inference_only_refit` results in significantly better quality and faster inference time.
+    medium_quality_fastest_train={'auto_stack': False},
+
+    # Optimizes result immediately for deployment by deleting unused models and removing training artifacts.
+    # Often can reduce disk usage by ~2-4x with no negatives to model quality or inference speed.
+    # This will disable numerous advanced functionality, but has no impact on inference.
+    # Recommended for applications where the inner details of AutoGluon's training is not important and there is no intention of manually choosing between the final models.
+    # This preset pairs well with the other presets such as `good_quality_faster_inference_only_refit` to make a very compact final model.
+    # Identical to calling `predictor.delete_models(models_to_keep='best', dry_run=False)` and `predictor.save_space()` directly after `fit()`.
+    optimize_for_deployment={'keep_only_best': True, 'save_space': True},
+
+    # Disables automated feature generation when text features are detected.
+    # This is useful to determine how beneficial text features are to the end result, as well as to ensure features are not mistaken for text when they are not.
+    ignore_text={'feature_generator_kwargs': {'enable_nlp_vectorizer_features': False, 'enable_nlp_ratio_features': False}},
+)
+
```

Comment:
Add TODO that in future should also consider configs with `hyperparameter_tune=True` (if training time doesn't matter, but inference-latency matters)

Reply:
Added